<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFrmParameterClient</class>
 <widget class="QWidget" name="CFrmParameterClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Key used to encrypt saved password:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Encrypt key:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leEncryptKey">
          <property name="inputMask">
           <string/>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbEncryptKey">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="eye-on">
            <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QRadioButton" name="rbPromptNo">
          <property name="text">
           <string>No</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbPromptFirst">
          <property name="text">
           <string>First</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbPromptAlways">
          <property name="text">
           <string>Always</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Prompt type:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Connecter name</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QCheckBox" name="cbShowPrefix">
        <property name="text">
         <string>Show protocol prefix in name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="cbShowIPPort">
        <property name="text">
         <string>Only show ip:port in name</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="cbHookKeyboard">
     <property name="text">
      <string>Hook: capture system key</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Password</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="cbSavePassword">
          <property name="text">
           <string>Save password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbEnableViewPassword">
          <property name="text">
           <string>Enable view password</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbHookKeyboard</tabstop>
  <tabstop>leEncryptKey</tabstop>
  <tabstop>pbEncryptKey</tabstop>
  <tabstop>rbPromptNo</tabstop>
  <tabstop>rbPromptFirst</tabstop>
  <tabstop>rbPromptAlways</tabstop>
  <tabstop>cbSavePassword</tabstop>
  <tabstop>cbEnableViewPassword</tabstop>
  <tabstop>cbShowPrefix</tabstop>
  <tabstop>cbShowIPPort</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
